### PR TITLE
Small documentation fix-ups for PINSFV

### PR DESCRIPTION
**Pull request by GiudGiud**
refs #16756

### CODE REVIEW

File: modules/navier_stokes/doc/content/modules/navier_stokes/fv.md [hunk 1]
```diff
@@ -1,6 +1,4 @@
-# Finite Volume Navier Stokes
-
-## Incompressible
+# Incompressible Finite Volume Navier Stokes
```
Comment (lindsayad) [inline modules/navier_stokes/doc/content/modules/navier_stokes/fv.md:1]: Why this change? Pretty soon there will be a compressible section. With this change we now jump from single `#` to triple `###`. I would say this is fine but then you should make everything else `##` ... but then when we add compressible I would immediately undo that change

File: modules/navier_stokes/doc/content/modules/navier_stokes/index.md [hunk 2]
```diff
@@ -9,7 +9,8 @@ For documentation specific to finite element or finite volume implementations,
 please refer to the below pages:
 
 - [Continuous Galerkin Finite Element](navier_stokes/cgfe.md)
-- [Finite Volume](navier_stokes/fv.md)
+- [Incompressible Finite Volume](navier_stokes/fv.md)
+- [Porous media Incompressible Finite Volume](navier_stokes/pinsfv.md)
```
Comment (lindsayad) [inline modules/navier_stokes/doc/content/modules/navier_stokes/index.md:13]: If you want to do this and your change above (with then changing `###` to `##`), then I think you should `git mv fv.md insfv.md`

Comment (lindsayad) [reply modules/navier_stokes/doc/content/modules/navier_stokes/index.md:13]: I would be supportive of that

**Review by lindsayad — CHANGES_REQUESTED** [review]

File: modules/navier_stokes/doc/content/modules/navier_stokes/index.md (hunk 2 above)
Comment (lindsayad) [reply modules/navier_stokes/doc/content/modules/navier_stokes/index.md:13]: This is what things look like in the `cnsfv-rebased` branch: [index.md](https://github.com/lindsayad/moose/blob/cnsfv-rebased/modules/navier_stokes/doc/content/modules/navier_stokes/index.md) and [fv.md](https://github.com/lindsayad/moose/blob/cnsfv-rebased/modules/navier_stokes/doc/content/modules/navier_stokes/fv.md)

Comment (lindsayad) [reply modules/navier_stokes/doc/content/modules/navier_stokes/index.md:13]: I'm going to add the link to `pinsfv.md` to `fv.md` right now as I'm doing the tabular breakdown of NS capabilities

Comment (lindsayad) [reply modules/navier_stokes/doc/content/modules/navier_stokes/index.md:13]: Actually yea I like the cutout of the middle-man `fv.md` and putting the links to `insfv.md, cnsfv.md, pinsfv.md` directly in `index.md` and then that allows us to do a tabular summary in `index.md` that includes both FE and FV

Comment (GiudGiud) [reply modules/navier_stokes/doc/content/modules/navier_stokes/index.md:13]: Ok I ll move fv.md to insfv.md and add that to the PR

File: modules/navier_stokes/doc/content/modules/navier_stokes/fv.md (hunk 1 above)
Comment (GiudGiud) [reply modules/navier_stokes/doc/content/modules/navier_stokes/fv.md:1]: Page was pretty long already I was thinking CNSFV could be its own page

File: modules/navier_stokes/doc/content/modules/navier_stokes/index.md (hunk 2 above)
Comment (GiudGiud) [reply modules/navier_stokes/doc/content/modules/navier_stokes/index.md:13]: umm you did it in CNSFV so I guess no need for me to do it

Comment (GiudGiud) [reply modules/navier_stokes/doc/content/modules/navier_stokes/index.md:13]: Ah I still need to change the ### to ##.
I ll do it soon

**Comment by GiudGiud** [comment]
good now?

**Review by lindsayad — APPROVED** [review]
Yea I think this is fine. This is probably going to give me a merge conflict but that's ok
